MTV-2230 Release Notes for 2.9.0 (#719)
kubev2v/forklift-documentation: documentation/modules/rn-2-9-0-resolved-issues.adoc

// Module included in the following assemblies:
//
// * documentation/doc-Release_notes/master.adoc

[id="resolved-issues-2-9-0_{context}"]
= Resolved issues 2.9.0

.{project-short} enabled vTPM for source Windows Server 2022 virtual machines without vTPM device
VMware only: In earlier releases of {project-short}, after warm migration of Windows Server 2022 VMs with UEFI settings, the virtual Trusted Platform Module (vTPM) device was added to the VMs even though the source VMs did not have the vTPM device. This issue has been resolved in {project-short} 2.9.0. link:https://issues.redhat.com/browse/MTV-2014[(MTV-2014)]

.{project-short} failed to start virtual machine with non-aligned disk size after migration 

> VMware only: In earlier releases of {project-short}, after migrating a VM with a disk size that did not align with the underlying storage, the VM failed to start. This issue has been resolved in {project-short} 2.9.0. link:https://issues.redhat.com/browse/MTV-2524[(MTV-2524)]

Reviewer: ```suggestion

VMware only: In earlier releases of {project-short}, after migrating a VM with a disk size that did not align with the underlying storage, the VM failed to start. This issue has been resolved in {project-short} 2.9.0. link:https://issues.redhat.com/browse/MTV-2524[(MTV-2524)]
```

.{project-short} does not refer to incorrect URL in VDDK error message
When you created a VMware provider with an incorrect VDDK URL and then created a migration plan, the plan displayed a `VDDK init image invalid` error message without a reference to the incorrect URL. This issue has been resolved in {project-short} 2.9.0. link:https://issues.redhat.com/browse/MTV-1150[(MTV-1150)]

.{project-short} swapped the MAC addresses after migrating VMware virtual machines with multiple NICs

> In earlier releases of {project-short}, warm and cold migrations of a VMware VM with multiple network interface cards resulted in swapped MAC addresses for the NICs, if the order of network mapping were different in the migration plan than the network mappings in the source VM. This issue has been resolved in {project-short} 2.9.0. link:https://issues.redhat.com/browse/MTV-2025[(MTV-2025)]

Reviewer: ```suggestion

In earlier versions of {project-short}, if the order of network mapping was different between the source VM and the migration plan, the MAC addresses of the network interface cards (NICs) could be switched during warm and cold migrations of a VMware VM with multiple NICs. This issue has been resolved in {project-short} 2.9.0.
```

.{project-short} failed OVA migration from unsupported source

> In earlier releases of {project-short}, when you try to import Open Virtual Appliance (OVA) file from a non-VMware source, the import failed due to Megabytes unit not being supported for the resource. This issue has been resolved in 2.9.0 by adding a warning about OVA from an unsupported source.
> link:https://issues.redhat.com/browse/MTV-2314[(MTV-2314)]

Reviewer: ```suggestion
In earlier releases of {project-short}, when you try to import an Open Virtual Appliance (OVA) file from a non-VMware source, the import failed due to Megabytes units not being supported for the resource. This issue has been resolved in 2.9.0 by adding a warning about OVA from an unsupported source. link:https://issues.redhat.com/browse/MTV-2314[(MTV-2314)]
```

//resolved or known issue with a workaround? //from here it's all non support escalation bugs
.{project-short} does not fully preserve VMware virtual machine UUID after migration
In earlier releases of {project-short}, the UUID of VMware VMs were not fully preserved after migration to {ocp-short}. This issue has been resolved through a warning in migration plans to inform you that VM UUIDs are truncated to 20 characters for `virtio` disks. link:https://issues.redhat.com/browse/MTV-1368[(MTV-1368)]

.{project-short} does not preserve host name of guest virtual machines after migration
In earlier releases of {project-short}, after migrating a VMware guest VM running RHEL OS, the host name configured in the `/etc/hostname` setting was not preserved in the target VM. This issue has been resolved in {project-short} 2.9.0. link:https://issues.redhat.com/browse/MTV-2364[(MTV-2364)]

.Cold migration of VMware ESXi virtual machines with IP address fail

> In earlier releases of {project-short}, cold migrations of ESXi VMs that were created with host IP address by using the vSphere provider failed because `virt-v2v` used the VM's host name instead of the IP address. This issue has been resolved in {project-short} 2.9.0. link:https://issues.redhat.com/browse/MTV-2153[(MTV-2153)]

Reviewer: ```suggestion

In earlier releases of {project-short}, cold migrations of ESXi VMs that were created with a host IP address by using the vSphere provider failed because `virt-v2v` used the VM's host name instead of the IP address. This issue has been resolved in {project-short} 2.9.0. link:https://issues.redhat.com/browse/MTV-2153[(MTV-2153)]
```

.{project-short} does not refer to incorrect URL in the VDDK error message
When you created a VMware provider with an incorrect VDDK URL and then created a migration plan, the plan displayed a `VDDK init image invalid` error message without a reference to the incorrect URL. This issue has been resolved in {project-short} 2.9.0. link:https://issues.redhat.com/browse/MTV-1150[(MTV-1150)]

.{project-short} swapped the MAC addresses after migrating VMware virtual machines with multiple NICs
In earlier versions of {project-short}, if the order of network mapping was different between the source VM and the migration plan, the MAC addresses of the network interface cards (NICs) could be switched during warm and cold migrations of a VMware VM with multiple NICs. This issue has been resolved in {project-short} 2.9.0. link:https://issues.redhat.com/browse/MTV-2025[(MTV-2025)]

.{project-short} failed OVA migration from unsupported source

> In earlier releases of {project-short}, when you try to import Open Virtual Appliance (OVA) file from a non-VMware source, the import failed due to Megabytes unit not being supported for the resource. This issue has been resolved in 2.9.0 by adding a warning about OVA from an unsupported source. link:https://issues.redhat.com/browse/MTV-2314[(MTV-2314)]

Reviewer: ```suggestion
In earlier releases of {project-short}, when you try to import an Open Virtual Appliance (OVA) file from a non-VMware source, the import failed due to Megabytes unit not being supported for the resource. This issue has been resolved in 2.9.0 by adding a warning about OVA from an unsupported source. link:https://issues.redhat.com/browse/MTV-2314[(MTV-2314)]
```

// Module included in the following assemblies:
//
// * documentation/doc-Release_notes/master.adoc

[id="resolved-issues-2-9-0_{context}"]
= Resolved issues 2.9.0

> .{project-short} enabled vTPM for source Windows Server 2022 virtual machines without vTPM device

Reviewer: ```suggestion
.{project-short} enabled vTPM for source Windows Server 2022 virtual machines without vTPM device

```

VMware only: In earlier releases of {project-short}, after warm migration of Windows Server 2022 virtual machines (VMs) with UEFI settings, the virtual Trusted Platform Module (vTPM) device was added to the VMs even though the source VMs did not have the vTPM device. This issue has been resolved in {project-short} 2.9.0. link:https://issues.redhat.com/browse/MTV-2014[(MTV-2014)]

> .{project-short} failed to start virtual machine with non-aligned disk size after migration 

Reviewer: ```suggestion
.{project-short} failed to start virtual machine with non-aligned disk size after migration

```

.{project-short} swapped the MAC addresses after migrating VMware virtual machines with multiple NICs
In earlier versions of {project-short}, if the order of network mapping was different between the source VM and the migration plan, the MAC addresses of the network interface cards (NICs) could be switched during warm and cold migrations of a VMware VM with multiple NICs. This issue has been resolved in {project-short} 2.9.0. link:https://issues.redhat.com/browse/MTV-2025[(MTV-2025)]

.{project-short} failed OVA migration from unsupported source
In earlier releases of {project-short}, when you try to import Open Virtual Appliance (OVA) file from a non-VMware source, the import failed due to Megabytes unit not being supported for the resource. This issue has been resolved in 2.9.0 by adding a warning about OVA from an unsupported source. link:https://issues.redhat.com/browse/MTV-2314[(MTV-2314)]

.{ocp-name} Container Platform to {ocp-name} Container Platform migration of VMware VM failed

> In earlier releases of {project-short}, after you successfully migrate a VMware virtual machine (VM) from {rhv-full} to an OCP cluster and then migrate the VM to another OCP cluster using {project-short}, the VM failed to start with a `No bootable device` error. This issue has been resolved in {project-short} 2.9.0 link:https://issues.redhat.com/browse/MTV-1544[(MTV-1544)]

Reviewer: ```suggestion

In earlier releases of {project-short}, after you successfully migrate a VMware virtual machine (VM) from {rhv-full} to an {ocp-short} cluster and then migrate the VM to another {ocp-short} cluster using {project-short}, the VM failed to start with a `No bootable device` error. This issue has been resolved in {project-short} 2.9.0 link:https://issues.redhat.com/browse/MTV-1544[(MTV-1544)]
```

//new entry
.{project-short} changed disk name of a virtual machine after migration from an OCP cluster to another OCP cluster
After you migrate a VMware guest VM to an OCP 4.18 cluster and then migrate that VM to another local OCP 4.18 cluster by using {project-short}, the disk names were changed. This issue has been resolved in {project-short} 2.9.0. link:https://issues.redhat.com/browse/MTV-2367[(MTV-2367)]

.{project-short} does not fully preserve VMware virtual machine UUID after migration
In earlier releases of {project-short}, the Universally Unique Identifiers (UUID) of VMware VMs were not fully preserved after migration to {ocp-short}. This issue has been resolved through a warning in migration plans to inform you that VM UUIDs are truncated to 20 characters for `virtio` disks. link:https://issues.redhat.com/browse/MTV-1368[(MTV-1368)]

.{project-short} does not preserve host name of guest virtual machines after migration

> In earlier releases of {project-short}, after migrating a VMware guest VM running Red Hat Enterprise Linux (RHEL), the host name configured in the `/etc/hostname` setting was not preserved in the target VM. This issue has been resolved in {project-short} 2.9.0. link:https://issues.redhat.com/browse/MTV-2364[(MTV-2364)]

Reviewer: ```suggestion

In earlier releases of {project-short}, after migrating a VMware guest VM running Red Hat Enterprise Linux (RHEL), the host name configured in the `/etc/hostname` setting was not preserved in the target VM. This issue has been resolved in {project-short} 2.9.0. link:https://issues.redhat.com/browse/MTV-2364[(MTV-2364)]
```

.Cold migration of VMware ESXi virtual machines with IP address fail

> In earlier releases of {project-short}, cold migrations of ESXi VMs that were created with a host IP address by using the vSphere provider failed because `virt-v2v` used the VM's host name instead of the IP address. This issue has been resolved in {project-short} 2.9.0. link:https://issues.redhat.com/browse/MTV-2153[(MTV-2153)]

Reviewer: ```suggestion

In earlier releases of {project-short}, cold migrations of ESXi VMs that were created with a host IP address by using the vSphere provider failed because `virt-v2v` used the VM's host name instead of the IP address. This issue has been resolved in {project-short} 2.9.0. link:https://issues.redhat.com/browse/MTV-2153[(MTV-2153)]
```
